site/src/pages/tutorials/tutorial-blog-rest-step3.mdx

```diff
@@ -0,0 +1,134 @@
+Now, map this service method with the HTTP POST method by adding the <type://@Post> annotation as follows.
+The endpoint to map is `/blogs`.
+
+```java filename=BlogService.java highlight=1
+@Post("/blogs")
+public HttpResponse createBlogPost(BlogPost blogPost) throws JsonProcessingException {}
+```
+
+## 2. Handle request parameters
+
+// TODO: TW 이 과정에서 할일 소개
+
+- [Assign a request converter](#assign-a-request-converter)
+- [Write a request converter](#write-a-request-converter)
+- [Store post information](#store-post-information)
+
+### Assign a request converter
+
+In this step, assign a request converter to the service method and then implement the converter function in the next step.
+Armeria's [request converter](../docs/server-annotated-service#converting-an-http-request-to-a-java-object) converts a client's request body into a Java object for you.
+
+Add the annotation <type://@RequestConverter> to the service method and specify the RequestConverterFunction class as BlogPostRequestConverter.class which we will implement in the next step.
+
+```java filename=BlogService.java highlight=1
+@Post("/blogs")
+@RequestConverter(BlogPostRequestConverter.class)
+public HttpResponse createBlogPost(BlogPost blogPost) throws JsonProcessingException {
+  
+}
+```
+
+### Write a request converter
+
+We can use Armeria's request converter as is, but here let's customize a request converter for our blog post requests.
+We want to convert blog post details into a Java object.
+Create a BlogPostRequestConvert.java file and write the following class in the file.
+
+1. Declare a class implementing the <type://RequestConverterFunction> interface. We use impromptu IDs in this tutorial.
+    ```java filename=BlogRequestConvert.java
+    final class BlogPostRequestConverter implements RequestConverterFunction {
+      private static final ObjectMapper mapper = new ObjectMapper();
+      private final AtomicInteger idGenerator = new AtomicInteger();
+    }
+    ```
+2. Write a method in the class to retrieve a value of a given key from a JSON object.
+
+  <CodeBlock language="java" filename="BlogRequestConverter.java" highlight="4">{`
+  static String stringValue(JsonNode jsonNode, String field) {
+    final JsonNode value = jsonNode.get(field);
+    if (value == null) {
+        throw new IllegalArgumentException(field + " is missing!");
+    }
+    return value.textValue();
+  }
+  `}</CodeBlock>
+3. Override the default `convertRequest()` method as follows.
+    - Map the `title` and `content` properties of the request body to a `BlogPost` object, using the `stringValue()` method we wrote in the previous step.
+    - Throw a <type://FallthroughException> at last, to try to convert other requests by the next converter.
+    - Feel free to add more mapping blocks in this request converter for other requests for your service.
+
+  <CodeBlock language="java" filename="BlogRequestConverter.java" highlight="9,10,13">{`
```

Comment:
![image](https://user-images.githubusercontent.com/25103250/112917902-9e9cad00-913e-11eb-8bfb-643861773b7e.png)
Thanks a lot for implementing this highlighting feature!
I think there should be some spaces between the line number and the code.
I think @trustin might help you with this. 😄

Reply:
Haha, I could really use @ikhoon 's help :rofl: